### PR TITLE
DG updates

### PR DESCRIPTION
**Pull request by PROGRAMMERHAO**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -266,6 +266,32 @@ _{Explain here how the data archiving feature will be implemented}_
...
+Also, our feature allows the user to input multiple tasks with the same descriptions and tags but with different names. This makes it easier for user to add repetitive tasks with similar details.
+
+You can find the specific implementation in the 'AddCommandParser' class and the 'AddCommand' class
+
+Given below is a scenario of how the add command is used and behaves.
+
+Step 1. The user inputs an 'add' Command with parameters 'n/CS2109S n/CS2103T n/homework'.
+
+Step 2. The AddCommandParser recognises that the input command has two names and one description as the parameters. Since there is no deadline prefix 'D' and event prefixes 'f' and 't', the parser will create tasks that are
+simple tasks. The tasks will have the same description but different names
```
Comment (lywich) [inline docs/DeveloperGuide.md:279]: I think this perhaps is a format error?

File: docs/images/AddCommand.png
Comment (lywich) [inline docs/images/AddCommand.png:1]: How about "until list size is 0" to be more explicit for the loop condition?

**Review by lywich — COMMENTED** [review]
Nice work as always, left some minor suggestions for you to consider.

File: docs/images/AddActivityDiagram.png
Comment (lywich) [inline docs/images/AddActivityDiagram.png:1]: Perhaps the conditions could be written in such a way like [if input contains deadline] and [else]...

**Review by lywich — COMMENTED** [review]
Nice work as always, left some minor suggestions for you to consider.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (PROGRAMMERHAO) [reply docs/DeveloperGuide.md:279]: yah thks!!

File: docs/images/AddCommand.png
Comment (PROGRAMMERHAO) [reply docs/images/AddCommand.png:1]: sure i will change it!

File: docs/images/AddActivityDiagram.png
Comment (PROGRAMMERHAO) [reply docs/images/AddActivityDiagram.png:1]: ok got it!!

**Review by joyngjr — APPROVED** [review]
LGTM! Thanks :)

**Pull request merged**